store/store.go

```diff
@@ -70,6 +74,20 @@ func (s *storeImpl) Get(key string) (interface{}, bool) {
 	return object.contents, true
 }
 
+// GetByRegex returns objects from the store based on a regexp search.
+func (s *storeImpl) GetByRegex(expr string) (m map[string]interface{}) {
+	s.mutex.RLock()
+	defer s.mutex.RUnlock()
+
+	m = make(map[string]interface{})
+	for k, v := range s.objects {
+		if matched, _ := regexp.MatchString(expr, k); matched {
```

Comment:
Good point, then it will match the behavior of `Get()` (which is what I was going for). I'll add this.